presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -75,6 +75,9 @@
     @Option(name = "--user", title = "user", description = "Username")
     public String user = System.getProperty("user.name");
 
+    @Option(name = "--password", title = "password", description = "Password")
```

Comment:
yup. That is in our backlog too :) But we were thinking it as a follow-on patch. Do you agree?